src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/rfc7240/long.polling.when.cache.is.empty.and.etag.is.latest/accept/client.rpt

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright 2016-2019 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property responseDate ${http_cache:date()} # sharedProperty
+property etag ${http_cache:strongEtag()} #shared
+property etag2 ${http_cache:strongEtag()} #shared
+
+connect await ROUTED_PROXY
+        "nukleus://streams/http-cache#0"
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:beginEx()
+                              .typeId(nukleus:id("http"))
+                              .header(":scheme", "http")
+                              .header(":method", "GET")
+                              .header(":path", "/data")
+                              .header(":authority", "localhost:8080")
+                              .header("prefer", "wait=20")
+                              .header("cache-control", "max-age=0")
+                              .header("if-none-match", etag2)
+                              .build()}
+connected
+
+write close
+
+read nukleus:begin.ext ${http:beginEx()
+                             .typeId(nukleus:id("http"))
+                             .header(":status", "304")
+                             .header("etag", etag2)
+                             .header("retry-after", "5")
+                             .build()}
+
+read closed
```

Comment:
If cache is empty and `etag` is latest, then is the `304` purely caused by timeout after `20s` ?

Reply:
I thought that you suggested we will send 304 if the second request goes to the origin server with if-none-match missing.

Reply:
Ah yes, now I recognize the rationale. 👍